lumigator/python/mzai/backend/pyproject.toml

```diff
@@ -28,7 +28,9 @@ dependencies = [
 
 [tool.uv]
 dev-dependencies = [
+    "alembic>=1.13.3",
     "pytest>=8.3.3",
+    "ruff>=0.7.0",
```

Comment:
Not 100% sure I need this one, open to suggestion/advice (and can remove and sort out the lock file too).